scripts/report_binary_size.js

```diff
@@ -0,0 +1,93 @@
+const { resolve } = require('path');
+const fs = require('fs');
+const { execSync } = require('child_process');
+
+const repoRoot = resolve(__dirname, '..');
+
+const commitHash = process.env.GITHUB_SHA || execSync('git rev-parse HEAD').toString();
```

Comment:
We seem to use `simple-git` for git operations in [other places in this repo](https://github.com/firebase/firebase-js-sdk/blob/master/scripts/release/utils/git.js#L18).

I'm okay with both. Will leave it up to you whether we want to pursue any consistency.